extensions/amp-story/1.0/amp-story-open-page-attachment.js

```diff
@@ -108,6 +133,71 @@ const renderDefaultPageAttachmentUI = (pageEl, attachmentEl) => {
   return openAttachmentEl;
 };
 
+/**
+ * Renders inline page attachment UI.
+ * @param {!Window} win
+ * @param {!Element} pageEl
+ * @param {!Element} attachmentEl
+ * @param {!Element} attachmentHref
+ * @return {!Element}
+ */
+const renderOutlinkPageAttachmentUI = (
+  win,
+  pageEl,
+  attachmentEl,
+  attachmentHref
+) => {
+  const openAttachmentEl = buildOpenOutlinkAttachmentElement(pageEl);
+
+  // Copy href to the element so it can be previewed on hover and long press.
+  if (attachmentHref) {
+    openAttachmentEl.setAttribute('href', attachmentHref);
+  }
+
+  // Setting theme
+  openAttachmentEl.setAttribute('theme', attachmentEl.getAttribute('theme'));
+
+  // Getting elements
+  const {chipEl, ctaLabelEl} = htmlRefs(openAttachmentEl);
+
+  // Appending text & aria-label.
+  const openLabelAttr = attachmentEl.getAttribute('data-cta-text');
+  const openLabel = openLabelAttr
+    ? openLabelAttr.trim()
+    : getLocalizationService(pageEl).getLocalizedString(
+        LocalizedStringId.AMP_STORY_PAGE_ATTACHMENT_OPEN_LABEL
+      );
+  ctaLabelEl.textContent = openLabel;
+  openAttachmentEl.setAttribute('aria-label', openLabel);
+
+  // Adding image.
+  const openImgAttr = attachmentEl.getAttribute('cta-image');
+
+  if (!attachmentEl.hasAttribute('cta-image') || openImgAttr !== 'none') {
```

Comment:
Use `openImgAttr` instead of `attachmentEl.hasAttribute('cta-img')`, if it's null it will be falsey. Also, why would someone use `none`?

Reply:
If they don't provide it, we default to a standard link icon. They have to specify "none" if they literally want to remove the image completely